client/state/selectors/get-current-locale-slug.js

```diff
@@ -1,24 +1,13 @@
 /**
- * Eternal dependencies
```

Comment:
This is a funny typo when you consider how hard it is to get rid of `lodash` once it's added to a project 😄